src/core/core.datasetController.js

```diff
@@ -978,6 +980,10 @@ class DatasetController {
 
 	/**
 	 * Utility for updating a element with new properties, using animations when appropriate.
+	 * @param {import("../core/core.element").default} element
```

Comment:
You can just use `Element` everywhere if you add a `typedef` to the top of the file, which would probably save a lot of characters

```
/**
 * @typedef { import("../core/core.element").default } Element
 */
```